src/ex2/index.html

```diff
@@ -1,24 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>Listy</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<div class="style">
+<head>
+    <title>Todo list Shirly</title>
     <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <div class="app-container">
-      <div class="list-container">
-        <h1 class="app-name">Todo App</h1>
-        <div class="list-controls">
-          <input
-            type="text"
-            id="list-item-input"
-            placeholder="Add your new todo"
-          />
-          <button type="button" id="list-item-submit">+</button>
-        </div>
-        <ul id="list"></ul>
-      </div>
+</head>
+<body>
+<div class="card">
+    <h1>Todo list</h1>
+    <div id="newToDo">
+        <input type="text" name="newToInput" id="newToInput" value="" placeholder="Add your new todo">
+        <button type="button" id="buttonAdd">+</button> 
     </div>
-  </body>
+    <div id="listToDo" ></div>
+    <div id="footer">
+        <h3 >You have <span class="lengthTodoList"> </span> pending tasks</h3>
```

Comment:
styling notes:
- when an element is unique and serves a specific purpose use the id attribute. There can only be 1 lengthTodoList element so id fits better here
- avoid spaces where not needed (<h3> instead of <h3 >)
- there is actually a <footer> tag you can use instead of a div